projects/sport-data-valley/seeds/questionnaires/mmm_questionnaire.rb

```diff
@@ -0,0 +1,508 @@
+              en: "How many times a week do you usually perform moderate physical activity for 30 minutes or more? (e.g. walking, cycling, rowing)" },
+    options: [
+      {
+        title: {  nl: "Niet",
+                en: "None" },
+        value: "none"
+      },
+      {
+        title: {  nl: "1 tot 2 keer per week",
+                en: "1 or 2 times a week" },
+        value: "1-or-2"
+      },
+      {
+        title: {  nl: "3 tot 4 keer per week",
+                en: "3 or 4 times a week" },
+        value: "3-or-4"
+      },
+      {
+        title: {  nl: "5 keer per week of vaker",
+                en: "5 times a week or more" },
+        value: "5-or-more"
+      },
+    ],
+    show_otherwise: false
+  },
+  {
+    id: 'study_motivation',
+    type: :range,
+    required: true,
+    section_start: {
+      nl: "Studiemotivatie",
+      en: "Study motivation"
+    },
+    section_end: true,
+    title: {  nl: "Op een schaal van 0 tot 10, waarbij 0 het laagst mogelijke en 10 het hoogst mogelijke is, geef aan hoe je jezelf scoort op je studiemotivatie:",
+              en: "On a scale of 0 to 10, where 0 is the lowest possible, and 10 is the highest possible score, please indicate how you rank yourself on your study motivation:" },
+    min: 0,
+    max: 10,
+    labels: [
+      {
+        nl: "laag",
+        en: "low"
+      },
+      {
+        nl: "hoog",
+        en: "high"
+      },
+    ],
+    step: 1
+  },
+  {
+    id: 'study_stress',
+    type: :range,
+    required: true,
+    section_start: {
+      nl: "Studiestress",
+      en: "Study stress"
+    },
+    section_end: true,
+    title: {  nl: "Op een schaal van 0 tot 10, waarbij 0 het laagst mogelijke en 10 het hoogst mogelijke is, geef aan hoe je jezelf scoort op de mate van ervaren stress door de studie:",
+              en: "On a scale of 0 to 10, where 0 is the lowest possible, and 10 is the highest possible score, please indicate how you rank yourself on the level of study related stress:" },
+    min: 0,
+    max: 10,
+    labels: [
+      {
+        nl: "laag",
+        en: "low"
+      },
+      {
+        nl: "hoog",
+        en: "high"
+      },
+    ],
+    step: 1
+  },
+  {
+    id: 'considered_discontinuing',
+    section_start: {
+      nl: "Intentie stoppen opleiding",
+      en: "Intention to stop training"
+    },
+    section_end: true,
+    type: :radio,
+    required: true,
+    title: {  nl: "Heb je in de afgelopen 7 dagen overwogen om te stoppen met je opleiding?",
+              en: "Have you seriously considered discontinuing your study during the last 7 days?" },
+    options: [
+      {
+        title: {  nl: "Ja",
+                en: "Yes" },
+        value: "yes"
+      },
+      {
+        title: {  nl: "Nee",
+                en: "No" },
+        value: "no"
+      },
+    ],
+    show_otherwise: false
+  },
+  {
+    #TODO Make sure the top is 10 and bottom is 0
```

Comment:
What is this?

Reply:
Not necessary anymore :sweat_smile: Forgot to remove the comment